### ISSUE TITLE
Basic auth is not supported

### ISSUE THREAD

**Issue by edwellbrook** [spolu/breach_core#71]
user should be prompted for username and password and instead this is ignored and access is denied.

more info http://tools.ietf.org/html/rfc2617

**Comment by spolu** [comment]
Indeed it's cruelly missing. Marked as dev.

**Comment by kesla** [comment]
Hey, this looks like fun. I'd be intersted in taking a look at how to get this working! Or has some progress already been started?

**Comment by spolu** [comment]
Not really. The API need to be added to the ExoBrowser: https://github.com/breach/exo_browser/issues/154
